base/src/lib.rs

```diff
@@ -5,3 +5,5 @@ pub mod hash;
 pub mod ipld;
 
 pub use cid;
+pub use multibase;
+pub use multihash;
```

Comment:
I think this is a good idea, it'll help with using the same versions downstream.